grizli/pipeline/default_params.py

```diff
@@ -14,7 +14,10 @@
 IR_M_FILTERS = ['F098M', 'F127M', 'F139M', 'F153M']
 IR_W_FILTERS = ['F105W', 'F110W', 'F125W', 'F140W', 'F160W']
 
-JW_FILTERS = ['F115W', 'F150W', 'F200W']
+NIRISS_FILTERS = ['F090W', 'F115W', 'F150W', 'F200W', 'F277W', 'F356W', 'F444W', 'F140M', 'F158M', 'F380M', 'F430M', 'F480M']
+NIRCAM_LW_FILTERS = ['F322W2', 'F277W', 'F356W', 'F444W', 'F250M', 'F300M', 'F335M', 'F360M', 'F410M', 'F430M', 'F460M', 'F480M', 'F323N', 'F405N', 'F466N', 'F470N']
+NIRCAM_SW_FILTERS = ['F150W2', 'F070W', 'F090W', 'F115W', 'F150W', 'F200W', 'F140M', 'F162M', 'F182M', 'F210M', 'F164N', 'F187N', 'F212N']
```

Comment:
This gets a little complicated with the filter names that are common to both NIRISS and NIRCam.  I guess `NIRCAM_SW_FILTERS + NIRCAM_LW_FILTERS`.  I think it's good to have them as separate lists, but `JW_FILTERS` should be just the unique values.

Reply:
Fixed here: 1b85be942c041dbd750a7037d0f4e5087e9190ff